real_time/surface_src/java_src/Alice/src/test/java/com/roboclub/robobuggy/main/RobobuggyConfigFileTest.java

```diff
@@ -0,0 +1,216 @@
+package com.roboclub.robobuggy.main;
+
+import com.roboclub.robobuggy.robots.PlayBackRobot;
+import com.roboclub.robobuggy.robots.SimRobot;
+import com.roboclub.robobuggy.robots.TransistorAuton;
+import com.roboclub.robobuggy.robots.TransistorDataCollection;
+import junit.framework.TestCase;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.io.*;
+
+/**
+ * Created by davidneiman on 9/2/16.
+ */
+public class RobobuggyConfigFileTest {
+
+    @Before
+    public void setUp() throws Exception {
+        //Runs before each test case, denoted "@Test" before the method
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        //Runs after each test case
+    }
+
+    public void createConfigFileWithRobot(String s) throws IOException{
+        BufferedWriter writer = new BufferedWriter(new FileWriter(new File("/Users/davidneiman/RoboBuggy/real_time/surface_src/java_src/Alice/config.config")));
+        writer.write("{\"COM_PORT_IMU\":\"NOT_SET\",\"COM_PORT_RBSM\":\"NOT_SET\",\"PORT_VISION\":\"NOT_SET\",\"COM_PORT_GPS\":\"NOT_SET\",\"WAYPOINT_SOURCE_LOG_FILE\":\"config.config\",\"PLAY_BACK_SOURCE_FILE\":\"NOT_SET\",\"VIDEO_FILE\":\"/Users/davidneiman/RoboBuggy/real_time/surface_src/java_src/Alice/logs/2016-02-20-06-50-45/webcam.mp4\",\"ROBOT_TYPE\":\"");
```

Comment:
I'd like it if you formatted this better, like

    "\{
        \"COM_PORT_IMU\":\"NOT_SET\",
        \"COM_PORT_RBSM\":\"NOT_SET\",
etc...